extensions/src/ACRE2DistortionTestPlugin/TsCallbacks.h

```diff
@@ -140,6 +143,12 @@ PLUGINS_EXPORTDLL void ts3plugin_onHotkeyEvent(const char* keyword);
 PLUGINS_EXPORTDLL void ts3plugin_onHotkeyRecordedEvent(const char* keyword, const char* key);
 PLUGINS_EXPORTDLL void ts3plugin_onClientDisplayNameChanged(uint64 serverConnectionHandlerID, anyID clientID, const char* displayName, const char* uniqueClientIdentifier);
 
+
+// API Compatibility
+// v23
+PLUGINS_EXPORTDLL void ts3plugin_onPluginCommandEventH();
```

Comment:
Oh btw, this is useless, I forgot to remove that after my first implementation idea ^^

Reply:
I was wondering what that was, but didn't bother trying to remove. :joy: Thanks.

Reply:
#696